pkg/service/cluster/service.go

```diff
@@ -385,6 +385,13 @@ func (s *Service) PutCluster(ctx context.Context, c *Cluster) (err error) {
 		)
 	}
 
+	//// Create the session and log error
+	_, err = s.GetSession(ctx, c.ID)
+	if err != nil {
+		s.logger.Info(ctx, "WARNING! Cannot create CQL session to the cluster. It will affect backup/restore/healthcheck services.",
+			"cluster_id", c.ID)
```

Comment:
It's a shame there is no `Warning` log level... 
Do you think that `Error` log level would fit better here? I don't have strong preferences, but it would make this more visible for sure.

Reply:
Info log is fine.